x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step_footer/create_step_footer.tsx

```diff
@@ -4,30 +4,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { FC, useState, useEffect } from 'react';
-import {
-  EuiCallOut,
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiProgress,
-  EuiSpacer,
-  EuiText,
-} from '@elastic/eui';
+import React, { FC, useEffect, useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { useMlKibana } from '../../../../../contexts/kibana';
+
 import {
   getDataFrameAnalyticsProgressPhase,
   DATA_FRAME_TASK_STATE,
 } from '../../../analytics_management/components/analytics_list/common';
 import { isGetDataFrameAnalyticsStatsResponseOk } from '../../../analytics_management/services/analytics_service/get_analytics';
+import { useMlKibana } from '../../../../../contexts/kibana';
 import { ml } from '../../../../../services/ml_api_service';
-import { DataFrameAnalyticsId } from '../../../../common/analytics';
+import { BackToListPanel } from '../back_to_list_panel';
+import { ViewResultsPanel } from '../view_results_panel';
+import { ProgressStats } from './progress_stats';
+import { ANALYSIS_CONFIG_TYPE } from '../../../../common/analytics';
 
 export const PROGRESS_REFRESH_INTERVAL_MS = 1000;
 
-export const ProgressStats: FC<{ jobId: DataFrameAnalyticsId }> = ({ jobId }) => {
+interface Props {
+  jobId: string;
+  jobType: ANALYSIS_CONFIG_TYPE;
+  showProgress: boolean;
+}
+
+export const CreateStepFooter: FC<Props> = ({ jobId, jobType, showProgress }) => {
   const [initialized, setInitialized] = useState<boolean>(false);
   const [failedJobMessage, setFailedJobMessage] = useState<string | undefined>(undefined);
+  const [jobFinished, setJobFinished] = useState<boolean>(false);
   const [currentProgress, setCurrentProgress] = useState<
     | {
         currentPhase: number;
```

Comment:
this should probably have a proper type which can also be used in `ProgressStats`, where currently `any` is currently being used.

Reply:
Updated in d12ebe435567047801b5da68847d3a54a520a0c5